### ISSUE TITLE
timed out waiting for page to load

### ISSUE THREAD

**Issue by snxwdrop** [Witness-senpai/fakku-downloader#25]
Tried downloading various mangas, still have Error: timed out waiting for page to load.

TY

**Comment by Benjki** [comment]
Did you even read the readme?

> Use option -w for set wait time between loading the pages. If quality of .png is bad, or program somewhere crush its can help.
> Use option -t for set timeout for loading first page.

**Comment by ajcolon48** [comment]
I think his cookie just expired.  Delete the cookies.pickle file and sign in again.

**Comment by snxwdrop** [comment]
> Did you even read the readme?
> 
> > Use option -w for set wait time between loading the pages. If quality of .png is bad, or program somewhere crush its can help.
> > Use option -t for set timeout for loading first page.

Yes I read it, if that has worked I did not even have to open my problem, anyway ty.

> I think his cookie just expired. Delete the cookies.pickle file and sign in again.

That did not worked.
